x-pack/plugins/spaces/public/copy_saved_objects_to_space/components/copy_to_space_flyout.tsx

```diff
@@ -25,7 +25,7 @@ import { ToastsStart } from 'src/core/public';
 import {
   ProcessedImportResponse,
   processImportResponse,
-} from '../../../../../../src/legacy/core_plugins/management/public';
+} from '../../../../../../src/legacy/core_plugins/kibana/public/management/sections/objects/lib/process_import_response';
```

Comment:
__question:__ is it possible to expose these types from `../../../../../../src/legacy/core_plugins/kibana/public` (i.e. add `index.ts` there)? This way we can at least track what legacy `kibana` types are still required by other plugins.

Reply:
good catch!